### PR TITLE
Rename custom_claims to custom_claims_buffer.

### PR DESCRIPTION
**Pull request by qiucwang**
As `custom_claims` in attest plugin now is a flat buffer, using plural is confusing.
This PR renamed `custom_claims` to `custom_claims_buffer`.

Signed-off-by: Qiucheng Wang <qiucwang@microsoft.com>

### CODE REVIEW

File: include/openenclave/internal/sgx/plugin.h [hunk 1]
```diff
@@ -72,9 +73,9 @@ oe_result_t oe_sgx_extract_claims(
  * @retval OE_INVALID_PARAMETER At least one parameter is invalid.
  * @retval An appropriate error code on failure.
  */
-oe_result_t oe_sgx_hash_custom_claims(
-    const void* custom_claims,
-    size_t custom_claims_size,
+oe_result_t oe_sgx_hash_custom_claims_buffer(
```
Comment (yentsanglee) [inline include/openenclave/internal/sgx/plugin.h:76]: Please rename parameter description at line 68 and 69.

Comment (qiucwang) [reply include/openenclave/internal/sgx/plugin.h:76]: Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for arbitrarily large thread-local data for SGX machines.
 
 ### Changed
+- Rename custom_claims to to custom_claims_buffer.
```
Comment (CodeMonkeyLeet) [inline CHANGELOG.md:18]:
```suggestion
- Rename the custom claims buffer added by oe_get_evidence from "custom_claims" to "custom_claims_buffer". Likewise, replace the `OE_CLAIM_CUSTOM_CLAIMS` definition for this name with `OE_CLAIM_CUSTOM_CLAIMS_BUFFER`.
```

Comment (qiucwang) [reply CHANGELOG.md:18]: Sorry for missing that. I'll update the changelog in next attestation PR which will be sent out soon.

**Pull request merged**